--- CMakeLists.txt	2013-01-15 22:38:32.000000000 +0900
+++ CMakeLists.txt	2013-08-15 17:53:41.253510947 +0900
@@ -148,7 +148,7 @@
 # HDF5 uses long long which is not ansi
 if (WIN32)
     # lots of warnings with cl.exe right now, use /W1
-    add_definitions("/W1 -D_CRT_SECURE_NO_WARNINGS -D_SCL_SECURE_NO_WARNINGS /bigobj")
+    # add_definitions("/W1 -D_CRT_SECURE_NO_WARNINGS -D_SCL_SECURE_NO_WARNINGS /bigobj")
 else(WIN32)
     add_definitions( "-Wall -Wno-unknown-pragmas -Wno-unused-function" )
 endif(WIN32)
